packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -215,4 +238,4 @@ export const VtexCommerce = (
       })
     },
   }
-}
+}
```

Comment:
Github is complaining about a new line at the end of file.
<img width="176" alt="Screenshot 2023-03-08 at 17 22 45" src="https://user-images.githubusercontent.com/11325562/223841268-691e1374-9f77-4286-8a19-bb898044380f.png">

```suggestion
}

```